<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>112</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>时间管理器</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>112</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabClock">
      <attribute name="title">
       <string>闹钟</string>
      </attribute>
      <widget class="QRadioButton" name="radioButton1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>铃声1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>铃声2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>铃声3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>选择铃声：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>定时时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="time">
           <time>
            <hour>9</hour>
            <minute>48</minute>
            <second>0</second>
           </time>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>301</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonSure">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStop">
          <property name="text">
           <string>试听/停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClear">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="labelShow">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间到了！</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>铃声名：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelSong">
          <property name="text">
           <string>请选择铃声</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSecond">
      <attribute name="title">
       <string>秒表</string>
      </attribute>
      <widget class="QPushButton" name="btnSecStop">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSecClear">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>141</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnSecStart_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>计时</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabCountDown">
      <attribute name="title">
       <string>倒计时</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请输入时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCdownStart">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCdownStop">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCdownClear">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelWaring">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入正确的时间格式！</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFinish">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>计时结束！</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>271</width>
         <height>35</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEditHour">
          <property name="font">
           <font>
            <family>Adobe Caslon Pro</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMin">
          <property name="font">
           <font>
            <family>Adobe Caslon Pro</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSec">
          <property name="font">
           <font>
            <family>Adobe Caslon Pro</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
